main.go

```diff
@@ -122,7 +122,10 @@ func main() {
 	fs.IntVar(&cli.SerfEventBuffer, "serf-event-buffer", 100000, "Size of serf 'recent event' buffer, outside of which things are dropped")
 	fs.IntVar(&cli.SerfMaxQueueDepth, "serf-max-queue-depth", 100000, "Size of Serf queue, outside of which things are dropped")
 	fs.StringVar(&cli.EnableAnalytics, "analytics", "disabled", "Enables analytics API: enabled or disabled")
-	fs.StringVar(&cli.AnalyticsMetricsURL, "analytics-metrics-url", "", "URL of Prometheus DB to store analytics data")
+	fs.StringVar(&cli.KafkaBootstrapServers, "kafka-bootstrap-servers", "", "URL of Kafka Bootstrap Servers")
+	fs.StringVar(&cli.KafkaUser, "kafka-user", "", "Kafka Username")
+	fs.StringVar(&cli.KafkaPassword, "kafka-password", "", "Kafka Password")
+	fs.StringVar(&cli.AnalyticsKafkaTopic, "analytics-kafka-topic", "", "Kafka Topic used to send analytics logs")
```

Comment:
If we want to use just a single kafka topic we should partition it by at least `EventType`+`CreatorID`.

Reply:
Good point. I guess it should be `session_id`, not creator_id, but yeat. Our initial idea with @ecmulli was to partition it only by `session_id`. This should be enough to sink data into Clickhouse. And if we plan to end up sending them to the same DB table, then do you see any value in partitioning (also) by `event.type`?

Reply:
For the context, for now we'll have 2 event types: `heartbeat` and `error`.

Reply:
I guess there will be some service reading it from kafka and writing into Clickhouse. If it gets stuck it'd better if it got stuck (because of e.g. some parsing issue) only for one type of event. Also some super-huge number of error messages shouldn't block processing heartbeats and the opposite.

Reply:
Yeah, it makes sense to me.

Reply:
Updated to use session_id+event_type as the kafka key.